### PR TITLE
Allow read_object_track_process to successfully configure

### PR DESCRIPTION
**Pull request by neal-siekierski**
`read_object_track_process` currently throws when its `reader` algorithm *passes* a nested configuration check. This PR negates the throw condition so that it throws when the check fails.

### CODE REVIEW

**Review by mwoehlke-kitware — APPROVED** [review]

**Review by dstoup — APPROVED** [review]
Is anything holding up a merge on this PR?

**Comment by neal-siekierski** [comment]
@dstoup, not to my knowledge. I'll merge it a bit later today unless something comes up.

**Pull request merged**
